js/app.js

```diff
@@ -25,13 +29,21 @@
    * on this dom element, set the className to the value of desiredClass
    */
 
+  function setMyLightClass(event, desiredClass) {
+    let lightId = this.dataset.lightId;
+    document.getElementById(lightId).className = desiredClass;
+  }
 
   /*
    * Declare 15 constants btn1, btn2, btn3, ...
    * use a dom element selector method to assign each const
    * to the button that has an id of the same name.
    */
 
+  let btn = [];
+  for (let i = 1; i <= 15; i++) {
```

Comment:
I don't think this works like you think it would. You'll probably need to write out each btn declaration in full.